### PR TITLE
feat: Add a confirmation to the down command

### LINKED ISSUE #151: Add a confirmation y/n question to the Nitric Down command
When the user performs destructive operations like Nitric down, we should give them that information and a chance to back out by selecting or pressing No.

### PR DESCRIPTION
**Pull request by asalkeld**
closes #151

### CODE REVIEW

File: pkg/cmd/stack/root.go [hunk 1]
```diff
@@ -213,6 +235,7 @@ func RootCommand() *cobra.Command {
 	cobra.CheckErr(stack.AddOptions(stackUpdateCmd, false))
 
 	stackCmd.AddCommand(stackDeleteCmd)
+	stackDeleteCmd.Flags().BoolVarP(&confirmDown, "yes", "y", false, "confirm the destruction of the stack")
```
Comment (HomelessDinosaur) [inline pkg/cmd/stack/root.go:238]: Would it be more intuitive to have the flag as --force/-f to destroy the stack unprompted?

Comment (jyecusch) [reply pkg/cmd/stack/root.go:238]: --force is probably better saved for highly destructive operations like deleting buckets containing files or databases.

**Review by jyecusch — APPROVED** [review]

File: pkg/cmd/stack/root.go (hunk 1 above)
Comment (raksiv) [reply pkg/cmd/stack/root.go:238]: Can we make this confirmation a little more verbose - 

"Warning - This operation will destroy your stack, all deployed resources will be removed. Are you sure you want to proceed?"

If possible, a better approach might be to show the things being destroyed

"Warning - the following resources will be destroyed:"
     Type                 Name            Plan
 -   Stack           quickstart-dev  delete
 -   ├─ aws:s3:BucketObject  index.html       delete
 -   └─ aws:s3:Bucket        my-bucket           delete

Comment (asalkeld) [reply pkg/cmd/stack/root.go:238]: personally think the above is a bit excessive. pulumi doesn't do that do they? I think in general cli tools don't overly baby the user too much. I'll update the text tho'

Comment (asalkeld) [reply pkg/cmd/stack/root.go:238]: raised https://github.com/nitrictech/cli/issues/168
For now i'll update the text

**Review by HomelessDinosaur — APPROVED** [review]

**Pull request merged**